test/e2e/suite/issuers/acme/certificate/webhook.go

```diff
@@ -180,7 +180,8 @@ func listOwnedChallenges(cl versioned.Interface, owner *cmacme.Order) ([]*cmacme
 	}
 
 	var owned []*cmacme.Challenge
-	for _, ch := range l.Items {
+	for i := range l.Items {
+		ch := l.Items[i]
```

Comment:
This one seems like a bug because the challenge pointer is appended to a list without DeepCopy.